File: cross-application-feature-development-management/Helpers/StringHelpers.cs
namespace cross_application_feature_development_management.Helpers
{
    public class StringHelpers
    {
        public string WrapInQuotationMarks(string value)
        {
            var valueToWrite = $"\"{value}\"";
            return valueToWrite;
        }
        public string StripQuotationMarks(string value)
        {
            var valueToWrite = value.Replace("\"", "");
            return valueToWrite;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Helpers/StringHelpers.fs
namespace cafdemwimu.console.Helpers

type StringHelpers() =
    member _.WrapInQuotationMarks(value: string) =
        let valueToWrite = $"\"{value}\""
        valueToWrite

    member _.StripQuotationMarks(value: string) =
        let valueToWrite = value.Replace("\"", "")
        valueToWrite
